packages/release/release-sysctl.conf

```diff
@@ -44,9 +44,6 @@ kernel.kptr_restrict = 1
 # Avoid kernel memory address exposures via dmesg.
 kernel.dmesg_restrict = 1
 
-# Turn off kexec, even if it's built in.
-kernel.kexec_load_disabled = 1
```

Comment:
What I'd like to do instead is:

1. start the system with kexec load enabled
2. (optionally) if we have a crash kernel reservation, load the kernel
3. set kexec_load_disabled = 1

Ideally this would happen in `preconfigured.target` so it runs before any container-supplied code.